webapp/components/about_build_modal.jsx

```diff
@@ -137,6 +137,13 @@ export default class AboutBuildModal extends React.Component {
                                     />
                                     {'\u00a0' + config.Version + '\u00a0' + config.BuildNumber}
                                 </div>
+                                <div>
+                                    <FormattedMessage
+                                        id='about.database'
```

Comment:
is this a new string?

if it is please add it to `webapp/i18n/en.json`.

Thanks